### ISSUE TITLE
Warn about too low value for `stream_select`'s `tv_usec` parameter

### ISSUE THREAD

**Issue by hollodotme** [kalessil/phpinspectionsea#1044]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Feature request                          |
| Plugin         | Php Inspections (EA Extended) |
| Language level | PHP4+                        |

Current behaviour (description/screenshot:)
--

When using the `stream_select` function it is [highly recommended by the official PHP documentation](http://php.net/stream_select), that the `tv_usec` parameter is set to at least `200000` when `tv_sec` is set to `0` to keep CPU usage at a reasonable level.

There is currently no inspection warning when using a lower `tv_usec` value.

<img width="723" alt="screen shot 2018-09-26 at 09 57 34" src="https://user-images.githubusercontent.com/1557839/46065550-a1377980-c172-11e8-94e6-badbf8f60774.png">

Expected behaviour
--

There should be a probable bug warning when the following conditions are met:

* `tv_sec` is set to `0`
* `tv_usec` is set to a value lower than `200000`

The warning message should contain a hint, that the given value may cause too high CPU usage.

The inspection should also apply if a static value in form of a constant is given to the function, like this:

```php
<?php declare(strict_types=1);

class MyStream
{
    private const TV_USEC = 20000;

    # ...

    public function hasResponse() : bool
    {
        $read = [$this->resource];
        $write = $except = [];
       
        stream_select($read, $write, $except, 0, self::TV_USEC);

        return 1 === count($read);
    }
}
```

Environment details
---
PhpStorm 2018.2.3 (but all versions)

**Comment by hollodotme** [comment]
Fine for me @kalessil. 😬

**Comment by kalessil** [comment]
Also a question: did you hit the issue in production (perhaps a ticket on GitHub)?

**Comment by hollodotme** [comment]
Yes, it was a long existing bug in [hollodotme/fast-cgi-client](https://github.com/hollodotme/fast-cgi-client/issues/20), that popped up during [this discussion](https://github.com/hollodotme/fast-cgi-client/issues/18) and was fixed in the [last release](https://github.com/hollodotme/fast-cgi-client/blob/v2.4.3/CHANGELOG.md).

It was quite hidden, because the `tv_usec` value was defined in a class constant. (That's why I mentioned the second part of this issue.) 😄
